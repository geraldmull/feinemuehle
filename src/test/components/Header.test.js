import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';



test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});

  
// test if there are one or more h1

// access to enzyme core api
// const wrapper = shallow(<Header />);

// expect(wrapper.find('h1').length).toBe(1);
// expect (wrapper.find('h1').text()).toBe('Header Expensify');