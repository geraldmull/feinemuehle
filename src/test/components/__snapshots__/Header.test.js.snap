// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Header Expensify App II
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    HOME
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    ADD EXPENSE
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    HELP
  </NavLink>
</header>
`;
